src/Angor/Client/Pages/Spend.razor

```diff
@@ -653,4 +688,31 @@
         await _clipboardService.WriteTextAsync(trxData);
         StateHasChanged();
     }
+    
+    public struct UtxoKey
+    {
+        public string Trxid { get; set; }
+        public int Outputindex { get; set; }
+
+        public UtxoKey(string trxid, int outputindex)
+        {
+            Trxid = trxid;
+            Outputindex = outputindex;
+        }
+
+        public override bool Equals(object obj)
+        {
+            if (!(obj is UtxoKey))
+                return false;
+
+            var key = (UtxoKey)obj;
+            return Trxid == key.Trxid && Outputindex == key.Outputindex;
+        }
+
+        public override int GetHashCode()
```

Comment:
ah I see you need to compile a hashcode for the dictionary unique key.
instead of using a new type it would be simpler to use a string of "trxid-index", but keep this as it is.